loopy/kernel/data.py

```diff
@@ -841,4 +844,47 @@ def __init__(self, target_name, result_dtypes, arg_dtypes):
 
 # }}}
 
+
+# {{{ Iname class
+
+class Iname(Taggable):
+    """
+    Records an iname in a :class:`~loopy.LoopKernel`.
```

Comment:
Add to docs.

Reply:
Included in `__doc__` at the top of the file. Anything else missing?